tests/integration/test_source_git.py

```diff
@@ -499,9 +499,9 @@ def test_srpm_merge_storm_dirty(api_instance_source_git):
     mock_spec_download_remote_s(sg_path, sg_path / DISTRO_DIR, ref)
     create_merge_commit_in_source_git(sg_path, go_nuts=True)
     (sg_path / "malt").write_text("Mordor\n")
-    with pytest.raises(PackitException) as ex:
-        with cwd("/"):  # let's mimic p-s by having different cwd than the project
-            api_instance_source_git.create_srpm(upstream_ref=ref)
+    with pytest.raises(PackitException) as ex, cwd("/"):
```

Comment:
that's 3.9? I thought this is a result of switching from LL(1) to PEG parser :eyes: and that's 3.10+ bugfix

_edit_: never mind, that was just for multi-line wrapped in parentheses…